src/components/ConfigureNewVault.tsx

```diff
@@ -114,12 +125,14 @@ const ConfigureNewVault = () => {
           label={toLockLabel}
           error={toLockError}
           actionLabel="Max"
-          onAction={onMaxClicked}
+          onAction={onMaxCollateralClicked}
         />
         <RatioPercentInput
-          onChange={setCollateralizationRatio}
+          onChange={() => {
+            /* always disabled */
+          }}
           value={collateralizationRatio}
-          disabled={!isInputReady}
+          disabled={true}
```

Comment:
I see this implements https://github.com/Agoric/dapp-inter/issues/73 as spec-ed, but I have doubts about the UX of an uneditable input field. Maybe we need a UI pattern for "dynamic value that you don't control". I see three kinds of numbers in the UI,
- user sets
- governance sets
- market sets
…and of course composites of those.

@otoole-brendan WDYT of having some visual queue to categorize them as such? If the categories were colors, the composite could be blends.

Reply:
@turadg not sure I follow what you mean by 'visual queue to categorize' - are you saying that anything a user sets is denoted by a global color, anything that is a governance parameter set (and not editable) is a different global color and market set i.e. ATOM price is a different global color again?

Reply:
By way of illustration, yes. Color is an example of a visual cue (misspelled earlier), which has the nice blending affordance, but I'm not suggesting that necessarily. Just that there be some consistent cue of what controls the number. Another could be,
- user numbers are in an input box
- governance numbers have a highlighted background
- market numbers have a little dot next to them to show how long ago they were read